model/adapter.py

```diff
@@ -723,6 +725,32 @@ def query_fun(rxn):
             reaction.bounds = 0, upper_bound
         return reaction
 
+    def minimize_distance(self):
+        """Replaces fluxomics measurements with the minimized distance"""
+        reaction_measurements = [m for m in self.measurements if m['type'] == 'reaction']
+        observations = {m['id']: np.mean(m['measurements']) for m in reaction_measurements}
+
+        uncertainties = {
+            m['id']: np.std(m['measurements']) if len(m['measurements']) >= 3 else 1
+            for m in reaction_measurements}
+
+        try:
+            # Include growth rate measurement
+            growth_rate = next(m for m in self.measurements if m['type'] == 'growth-rate')
```

Comment:
Neither is very pretty but an alternative here could be the use of `next(<generator>, None)` combined with checking `if growth_rate is not None`.

Reply:
@Midnighter would it make sense to stop accepting `growth-rate` as a separate measurement type, and instead send it like a normal reaction measurement with the biomass reaction id? That would require adding [the biomass id to species mapping](https://github.com/DD-DeCaF/model/blob/devel/model/constants.py#L129) in `iloop-to-model`, though. WDYT?

Reply:
I imagine `"growth-rate"` is important in other places such as creating a warning and for the interactive map legend?

Reply:
No, this is just about the measurement data sent to the model service for data-adjusted. The value is not used beyond the flux calculation